test/userapp/index.js

```diff
@@ -398,7 +441,12 @@ const addRequestHeader = (request, z, bundle) => {
   could be useful for APIs that always return 200 even on errors.
  */
 const changeStatusOnErrorResponses = (response) => {
-  const isJsonResponse = response.getHeader('Content-Type').match(/^application\/json/);
+  const contentType = response.getHeader('Content-Type');
+  if (!contentType) {
+    return response;
+  }
+
+  const isJsonResponse = contentType.match(/^application\/json/);
```

Comment:
Decided to keep what we had since 1) it is test-only code so not too worried of any performance impacts 2) It reads about the same once you have `indexOf('application/json') === 0`